### DISCUSSION TITLE
swag config

### DISCUSSION THREAD

**Discussion by luckyzor** [jlesage/docker-jdownloader-2#204] [Q&A]
Hello,

I'm new to swag and jdownloader 2, can't find any config for swag.
can anyone help me?
I tried but nothing works.

thank you

**Comment by bitdruid** [comment]
> Hello,
> 
> I'm new to swag and jdownloader 2, can't find any config for swag. can anyone help me? I tried but nothing works.
> 
> thank you

can you specify what you exactly trying to achive? do you mean reverse-proxy configuration?

**Reply by luckyzor** [reply]
Yes, exactly.

**Reply by luckyzor** [reply]
But adapted for swag
